verilog/tools/ls/README.md

```diff
@@ -236,3 +271,55 @@ code --install-extension verible.vsix
 ```
 
 [release]: https://github.com/chipsalliance/verible/releases
+
+## Configuring the Language Server for a project
+
+### Adding `verible.filelist` for project-wide symbol discovery
+
+`verible-verilog-ls` by default loads and analyses only currently edited files in the editor.
+To be able to utilize such features as going to definition or references project-wide, add a `verible.filelist` file to the project.
+`verible.filelist` is a file containing a list of design files to collect data necessary to build a symbol table and other helper structures to allow Language Server to provide definitions, references, information on hover and more.
+It is a simple file containing one file per line - it can be easily created using such tools as `find`, e.g.:
+
+```bash
+find . -name "*.sv" -o -name "*.svh" -o -name "*.v" | sort > verible.filelist
+```
+
+With it, the Language Server creates a symbol table for the entire project, allowing to find definitions and references for a symbol accross multiple files.
+
```

Comment:
Important: where is the file located ? We need to describe what we look for and what that means. Toplevel can come from the editor, if it provides a root.uri (and so here you need to find what this is commonly called in these editors). Or we fall back to the current working directory the editor has been opened in (but: what does that mean for Windows, where people just click on an icon somewhere and the root directory is not so clearly defined ?).

So here, we need to explain where it is and potentially figuring out with various edtiors what that means in their context.

Maybe we need a separate paragraph about the root dir situation, as then we can refer to it in the context of the file list and the linter configuration.

Reply:
I added some description regarding the workspace directory and where it can be located